app/controllers/v0/sessions_controller.rb

```diff
@@ -99,11 +99,14 @@ def user_login(saml_response)
       if user_session_form.valid?
         @current_user, @session_object = user_session_form.persist
         set_cookies
-        # track users who need to re-login on MHV
-        StatsD.increment(STATSD_MHV_COOKIE_NO_ACCOUNT_KEY) unless @current_user.mhv_correlation_id
         after_login_actions
         redirect_to url_service.login_redirect_url
-        stats(:success, saml_response)
+        if self.location.start_with?(url_service.base_redirect_url)
```

Comment:
Interesting is self.location the value of the `Location` header that will get sent in the redirect_to response above?

Reply:
yeah https://github.com/rails/rails/blob/be3d9daaa39aa69603af16b602fe62ae47548469/actionpack/lib/action_controller/metal/redirecting.rb#L63